### DISCUSSION TITLE
Observations on Open-Source Software Monetization and Limitations

### DISCUSSION THREAD

**Discussion by maxsonaraujo** [RocketChat/Rocket.Chat#30186] [General]
It's okay to have an enterprise version and a free version based on an open-source project. It makes sense for the enterprise version to have specific and improved features for a different cost. However, what doesn't make sense is to create limitations like the number of users, for example, in the free option. Another thing that doesn't make sense is limiting the number of installations in the marketplace. It would make sense to have costs for plugins or to charge a fee for those who publish paid plugins. Charging for hosting and support would also make sense. Among all these considerations, what shocked me the most was the open-source version having a user account limit for self-hosted setups. I understand they need ways to generate revenue, but based on the points I mentioned earlier. However, this aspect seems unfair; the basic essence of a chat application is its users, in addition to the conversations.

**Comment by rusekr** [comment]
Yes, It shocked me too. And limiting number of departments on omnichannel even for older versions. Just thinking about alternatives because who knows what developers cut in next version.

**Reply by fullheart** [reply]
Yes, these limitations are hard to understand and the stability is getting worse too. So also the Blender Foundation organization is leaving Rocket.Chat and moving to [Element](https://element.io/) (Matrix). See announcement from August 9th, 2024: https://devtalk.blender.org/t/blender-chat-migration
